docs/Customization/Memory-Hierarchy.rst

```diff
@@ -16,6 +16,12 @@ configure 4 KiB direct-mapped caches for L1I and L1D.
 
     import freechips.rocketchip.subsystem.{WithNMediumCores, WithNSmallCores}
 
+    class SmallRocketConfig extends Config(
+        new WithTop ++                                       // use default top
+        new WithBootROM ++                                   // use default bootrom
+        new freechips.rocketchip.subsystem.WithNSmallCores(1) ++
+        new freechips.rocketchip.system.BaseConfig)
+
     class SmallRocketConfig extends Config(
         new WithNSmallCores(1) ++
         new RocketConfig)
```

Comment:
Since this is a duplicate/broken config, lets delete this.

Reply:
Also, I can't comment on `MediumRocketConfig` in the GitHub GUI, but was that config passing? If not, could you fix it to be like the `SmallRocketConfig` (but just with `Medium` cores).

Reply:
Quickly looking at Rocket Chip, it looks like `WithNMediumCores` should instead be `WithNMedCores`.

Reply:
This and the medium config do work fine on their own, just not when used as part of the scratchpad setup. In my most recent commit, I renamed the config used for the scratchpad as `SmallRocketConfigNoL2`. If you'd rather still completely remove the small/medium configs at the top, let me know!